moped-etl/arcgis/cli.py

```diff
@@ -28,10 +28,10 @@ def get_cli_args():
     )
 
     parser.add_argument(
-        "-t",
-        "--test",
+        "-n",
+        "--dry-run",
         action="store_true",
-        help="Run script without making changes to the AGOL dataset",
+        help="Log what changes would be made without executing them",
```

Comment:
This diff formatting makes it look like I thought -- hey, i want to change the name of this flag for no good reason. 

In actuality, I wanted to vibe-code a diff on an arbitrary ETL to show what adding a `--dry-run` ability would look like so we could look at it in a meeting. I figured I'd be pitching the PR after that, but @mddilley expressed interest in looking at it, graciously, so I came back through to spiff it up for review.

Anyway - I picked this ETL at semi-random, and it just so happens to have ALREADY had a dry-run like feature in it. So when I added a new one, we had two, so I stripped back out `--test` so that I could offer this PR as a ever-slightly-marginal improvement in the logging, but obv, no major functionality ended up being added.